service-sdk/12.0.0/migrate-to-open-api/messaging-api/pom.xml

```diff
@@ -11,12 +11,12 @@
 
     <groupId>com.backbase.message</groupId>
     <artifactId>messaging-api</artifactId>
-    <version>1.0.0-SNAPSHOT</version>
+    <version>1.0.0</version>
     <name>Backbase :: messaging-api</name>
 
     <properties>
         <!-- plugin versions -->
-        <boat-maven-plugin.version>0.13.0</boat-maven-plugin.version>
+        <boat-maven-plugin.version>0.14.0</boat-maven-plugin.version>
```

Comment:
This POM illustrates what OpenAPI archetype 12.1.0 actually generates so should probably stay as 0.13.0 here.